genesyscloud/tfexporter/resource_genesyscloud_tf_export.go

```diff
@@ -114,17 +111,17 @@ func ResourceTfExport() *schema.Resource {
 				Elem:        &schema.Schema{Type: schema.TypeString},
 				ForceNew:    true,
 			},
+			"add_depends_on_capability": {
+				Description: "Add Depends On Capability for Flows",
```

Comment:
Replace with: `Adds a "depends_on" attribute to genesyscloud_flow resources with a list of resources that are referenced inside the flow configuration.`